Public/External/BuildPrediction/BuildPredictionTests/StandardPredictors/IntermediateOutputPathIsOutputDirTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.Build.Construction;
+using Microsoft.Build.Evaluation;
+using Microsoft.Build.Execution;
+using Microsoft.Build.Prediction.StandardPredictors;
+using Xunit;
+
+namespace Microsoft.Build.Prediction.Tests.StandardPredictors
+{
+    public class IntermediateOutputPathIsOutputDirTests
+    {
+        static IntermediateOutputPathIsOutputDirTests()
+        {
+            MsBuildEnvironment.Setup(TestHelpers.GetAssemblyLocation());
+        }
+
+        [Fact]
+        public void IntermediateOutputPathFoundAsOutputDir()
+        {
+            const string IntermediateOutputPath = @"C:\repo\bin\x64";

```

Comment:
>C:\repo\bin\x64 [](start = 52, length = 15)

nit: Consider 1 more case with relative path to cover both parts of the rooted path check in the predictor

Reply:
will do